spec/requests/checkout/discounts_spec.rb

```diff
@@ -155,7 +155,7 @@
         fill_in "Quantity", with: "10"
 
         check "Limit validity period"
-        fill_in "Valid from", with: "11-02-2022\t11:00PM"
+        fill_in_datetime "Valid from", with: "2022-11-02T23:00"
```

Comment:
Changed date format from "11-02-2022\t11:00PM" to "2022-11-02T23:00" for Inertia/React migration. The old format worked with the legacy server-side date picker, but React's DateInput uses HTML5 datetime-local inputs which require ISO8601 format.